Add section on how to handle Glitch node limits (#32695)
mdn/content: files/en-us/learn/server-side/express_nodejs/deployment/index.md

- Select GitHub in the popup to sign up using your GitHub credentials.
- You'll then be logged in to the Glitch dashboard: <https://glitch.com/dashboard>.

> ### Update Node Versions

Reviewer: Is this a "Troubleshooting" section, maybe? Common pitfalls?

```suggestion
### Troubleshooting Node.js version
```

> Hosting providers commonly support some major version of recent node releases.

Reviewer: ```suggestion
Hosting providers commonly support some major version of recent Node.js releases.
```

If the exact "minor" version you have specified in your `package.json` file is not supported they will usually fall back to the closest version they support (and often this will just work).

> Unfortunately, at time of writing, the highest supported version on Glitch is node 16.

Reviewer: ```suggestion
Unfortunately, at time of writing, the highest supported version on Glitch is Node.js 16.
```

> If you have been developing with node 17 or later, you should reduce the version used in your `package.json` file as shown.

Reviewer: ```suggestion
If you have been developing with Node.js 17 or later, you should reduce the version used in your `package.json` file as shown.
```

You will also need to retest:

```json
  "engines": {
    "node": ">=v16"
  },
```

> Glitch [plans to update node and keep it better updated in future](https://blog.glitch.com/post/rebuilding-glitch) — and it may be that by the time you read this the node version limit no longer exists.

Reviewer: ```suggestion
Glitch [plans to update node and keep it better updated in future](https://blog.glitch.com/post/rebuilding-glitch) — and it may be that by the time you read this the version limit no longer exists.
```

> Instead of updating node for your project first, you might upload it and see if it works.
> If the application doesn't load you could then modify the `package.json` in the Glitch editor to reduce the node version.

Reviewer: ```suggestion
Instead of downgrading the `node` version, you could upload your project to see if it builds.
If there are errors and your application doesn't load, you should try setting the `node` version to `>=v16` in your `package.json` in the Glitch editor.
```
